library/core/src/ops/arith.rs

```diff
@@ -612,6 +642,13 @@ pub trait Neg {
     type Output;
 
     /// Performs the unary `-` operation.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// let x: i32 = 12;
+    /// assert_eq!(-x, -12);
```

Comment:
I was going to say something about `assert_eq!(-12, -12)` and then I realized that was silly lol. So this is fine.

Reply:
:D

Reply:
These suggestions are about the _meaning_ of the operations, you're showing off the edge cases. I don't think this is a good place to document edge cases, this is just showing which operations the trait corresponds to.

Reply:
In particular, all these suggestions only make sense for numbers and would be completely out of place if it used `Duration` or something instead.

Reply:
Well, edge cases are considered part of the meanings too, of course users wouldn't want surprises. Or maybe do you have a better idea where to put these edge cases?

Reply:
I'd document them on the _implementations_, not the traits themselves. Take a look at `library/core/src/num`, maybe. But they certainly don't belong here - this is for _every_ type that implements the trait, not just numbers.